examples/file-upload-download/public/index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>File upload and download</title>
+
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      rel="shortcut icon"
+      type="image/x-icon"
+      href="https://loopback.io/favicon.ico"
+    />
+
+    <style>
+      h3 {
+        margin-left: 25px;
+        text-align: center;
+      }
+
+      a,
+      a:visited {
+        color: #3f5dff;
+      }
+
+      h3 a {
+        margin-left: 10px;
+      }
+
+      a:hover,
+      a:focus,
+      a:active {
+        color: #001956;
+      }
+
+      .power {
+        position: absolute;
+        bottom: 25px;
+        left: 50%;
+        transform: translateX(-50%);
+      }
+
+      .info {
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+      }
+
+      .info h1 {
+        text-align: center;
+        margin-bottom: 0;
+      }
+
+      .info p {
+        text-align: center;
+        margin-bottom: 3em;
+        margin-top: 1em;
+      }
+    </style>
+
+    <script>
+      /**
+       * Submit the upload form
+       */
+      function setupUploadForm() {
+        const formElem = document.getElementById('uploadForm');
+        formElem.onsubmit = async e => {
+          e.preventDefault();
+
+          const res = await fetch('/file-upload', {
+            method: 'POST',
+            body: new FormData(formElem),
+          });
+
+          const body = await res.json();
+          console.log('Response from upload', body);
+
+          await fetchFiles();
+        };
+      }
+
+      /**
+       * List uploaded files
+       */
+      async function fetchFiles() {
+        const res = await fetch('/file-download');
+        const files = await res.json();
+        console.log('Response from list', files);
+        const list = files.map(
+          f => `<li><a href="/file-download/${f}">${f}</a></li>\n`,
+        );
+        document.getElementById('fileList').innerHTML = list.join('');
+      }
+
+      async function init() {
+        setupUploadForm();
+        await fetchFiles()
+      }
+
+    </script>
+  </head>
+
+  <body onload="init();">
+    <div class="info">
+      <h1>File upload and download</h1>
+
+      <div id="upload">
+        <h3>Upload files</h3>
+        <form id="uploadForm">
+          <label for="files">Select files:</label>
+          <input type="file" id="files" name="files" multiple /><br /><br />
+          <label for="note">Note:</label>
+          <input
+            type="text"
+            name="note"
+            id="note"
+            value="Note about the files"
```

Comment:
Can you change `value` to `placeholder`?
Also, is it possible to show the note we put here beside the name of the file in the main page?

Reply:
The fields is printed out in console.log of the browser.